app/scripts/modules/core/src/pipeline/config/triggers/pubsub/pubsub.trigger.ts

```diff
@@ -1,19 +1,29 @@
 import { IController, module } from 'angular';
 
 import { PIPELINE_CONFIG_PROVIDER, PipelineConfigProvider } from 'core/pipeline/config/pipelineConfigProvider';
-import { IPubsubTrigger } from '@spinnaker/core';
+import {
+  IPubsubTrigger,
+  PUBSUB_SUBSCRIPTION_SERVICE,
+  PubsubSubscriptionService,
+} from '@spinnaker/core';
 
 class PubsubTriggerController implements IController {
   public pubsubSystems = ['kafka', 'google'];
+  public pubsubSubscriptions: string[];
 
-  constructor(public trigger: IPubsubTrigger) {
+  constructor(public trigger: IPubsubTrigger,
+              public pubsubSubscriptionService: PubsubSubscriptionService) {
```

Comment:
Should be `private`.

Reply:
`private` throws a linter error, I just removed `public`.